biome.json

```diff
@@ -0,0 +1,15 @@
+{
+	"formatter": {
+		"ignore": ["website/**", "packages/*/dist/**"]
+	},
+	"javascript": {
+		"formatter": {
+			"quoteStyle": "single",
+			"semicolons": "always",
+			"trailingComma": "all",
+			"lineWidth": 120,
+			"indentStyle": "space",
```

Comment:
Will just drop the bomb here, but could we maybe switch to tabs? Leaving aside the neverending conversations that happen when the question is asked, I'm mostly asking for accessibility reasons (someone might have a harder time reading an indent of 4 for example, and want to make their editor render it smaller or larger) and tabs would mean being able to just make their editor show that, without affecting the indentation in the code itself 👀

Reply:
@vladfrangu chose violence on Friday :D

Reply:
I like tabs, always missed them when I switched to JS and was forced to use two spaces to be consistent :D 

And yes, since we are adopting a formatter, this should be a single toggle switch for us, so I would be actually in favor of doing that.

Reply:
As long as the style is enforced with a formatter, I'm down with anything :slightly_smiling_face: